core/src/main/java/info/nightscout/androidaps/interfaces/BgSourceInterface.java

```diff
@@ -9,4 +9,6 @@
     boolean advancedFilteringSupported();
 
     void handleNewData(Intent intent);
+
+    int getSensorBatteryLevel();
```

Comment:
I'd prefer default function here instead of override everywhere and return -1

Reply:
That was my first idea but abstract method cannot have body...
I have to add a dedicated module like PumpDescription ?